source/lex.tex

```diff
@@ -464,22 +464,22 @@
 
 \begin{tokentable}{Alternative tokens}{lex.digraph}{Alternative}{Primary}
 \tcode{<\%}             &   \tcode{\{}         &
-\tcode{and}             &   \tcode{\&\&}       &
-\tcode{and_eq}          &   \tcode{\&=}        \\ \rowsep
+\keyword{and}           &   \tcode{\&\&}       &
+\keyword{and_eq}        &   \tcode{\&=}        \\ \rowsep
 \tcode{\%>}             &   \tcode{\}}         &
-\tcode{bitor}           &   \tcode{|}          &
-\tcode{or_eq}           &   \tcode{|=}         \\ \rowsep
+\keyword{bitor}         &   \tcode{|}          &
+\keyword{or_eq}         &   \tcode{|=}         \\ \rowsep
 \tcode{<:}              &   \tcode{[}          &
-\tcode{or}              &   \tcode{||}         &
-\tcode{xor_eq}          &   \tcode{\caret=}    \\ \rowsep
+\keyword{or}            &   \tcode{||}         &
+\keyword{xor_eq}        &   \tcode{\caret=}    \\ \rowsep
 \tcode{:>}              &   \tcode{]}          &
-\tcode{xor}             &   \tcode{\caret}     &
-\tcode{not}             &   \tcode{!}          \\ \rowsep
+\keyword{xor}           &   \tcode{\caret}     &
+\keyword{not}           &   \tcode{!}          \\ \rowsep
 \tcode{\%:}             &   \tcode{\#}         &
-\tcode{compl}           &   \tcode{\~}         &
-\tcode{not_eq}          &   \tcode{!=}         \\ \rowsep
+\keyword{compl}         &   \tcode{\~}         &
+\keyword{not_eq}        &   \tcode{!=}         \\ \rowsep
 \tcode{\%:\%:}          &   \tcode{\#\#}       &
-\tcode{bitand}          &   \tcode{\&}         &
+\keyword{bitand}        &   \tcode{\&}         &
```

Comment:
These are not keywords.

Reply:
They're close enough to being keywords for the purposes of indexing and highlighting, I think.